src/common/constants/complexLookup.constants.ts

```diff
@@ -0,0 +1,38 @@
+import { SEARCH_API_ENDPOINT } from './api.constants';
+
+export const COMPLEX_LOOKUPS_CONFIG: ComplexLookupsConfig = {
+  authorities: {
+    api: {
+      endpoint: `${SEARCH_API_ENDPOINT}/authorities`,
+      searchQuery: {
+        filter: '(type <> "CONCEPT")',
+      },
+    },
+    labels: {
+      button: {
+        base: 'marva.assignAuthority',
+        change: 'marva.change',
+      },
+      modal: {
+        title: {
+          creator: 'marva.searchCreatorAuthority',
+          contributor: 'marva.searchCreatorAuthority',
+        },
+        searchResults: 'marva.authorities',
+      },
+    },
+    // For displaying "Search by" control
+    searchBy: [
+      {
+        label: 'keyword',
+        value: 'label',
+      },
+    ],
+    linkedFields: ['subclass'],
+    customFields: {
+      source: {
+        label: 'marva.lcnaf',
```

Comment:
Hi @SKarolFolio, As I mentioned in my comment in the ticket, it will be incorrect to hardcode "LCNAF" in the source column, beaus there will be authority records sourced from both Bib records and LCNAF. I think it is better to remove "source" column for beta. We can discuss with Doug tomorrow.

Reply:
Commented Source column and the related code.
Will update or remove in the next PR after discussion.